content/docs/release-notes/22-10-0/_index.md

```diff
@@ -0,0 +1,271 @@
+
+  For any other project, one person must convert the assets by following these steps:
+
+  1. Run Editor. Do not open any level, just close the "Welcome to O3DE" screen.
+  1. Wait until Asset Processor has finished processing all the assets.
+  1. Open Console by clicking **Tools** > **Console**.
+  1. Enter the following console command to convert old physics material libraries: `ed_physxConvertMaterialLibrariesIntoIndividualMaterials`
+  1. Wait until Asset Processor has finished processing all the assets.
+  1. Enter the following console command to fix prefabs and fbx manifests that reference old assets: `ed_physxFixAssetsUsingPhysicsLegacyMaterials`
+  1. Wait until Asset Processor has finished processing all the assets.
+  1. Commit the source asset changes into Github.
+
+**Viewport**
+
+- The Component Mode Switcher may suffer from instability on Linux. If this is experienced you can disable the Switcher in the Settings Registry by adding the following: 
+
+  ```
+  {
+      "Amazon": {
+          "Preferences": {
+              "Editor": {
+                  "ComponentSwitcherEnabled": false
+              }
+          }
+      }
+  }
+  ```
+
+**Prefab**
+
+- After creating and deleting prefabs, you must save your level twice.
+
+- After prefab operations, you may encounter warnings that say "Some of the patches were not successfully applied". 
+  - For prefab deletion, it has been fixed in development. [#11756](https://github.com/o3de/o3de/issues/11756)
+  - For prefab creation, it will be fixed in development. [#12013](https://github.com/o3de/o3de/issues/12013)
+
+- Deleting or reordering a nested prefab in Prefab Edit Mode disrupts the rest of the outliner view. [#11737](https://github.com/o3de/o3de/issues/11737)
+
+**Editor**
+
+- On Linux, the O3DE Editor splash screen displays "Development Build" instead of the version number if development and release are installed together. [#9519](https://github.com/o3de/o3de/issues/9519)
+
+- There's an issue with running the Game Launcher in a maximized window when the user presses "Play Game (Maximized)" in the O3DE Editor. [#12125](https://github.com/o3de/o3de/pull/12125)
+
+- For projects with all Gems enabled, if a user previously closed the O3DE Editor while it was still processing assets, the O3DE Editor crashes on splash screen when it's reopened. [#11926](https://github.com/o3de/o3de/issues/11926) This issue will be resolved in development: [#12445](https://github.com/o3de/o3de/pull/12445)
+
+- Editor crashes in `AZ::TraskGraphEvent::Signal` occasionally. [#12015](https://github.com/o3de/o3de/issues/12015)
+
+**Script Canvas**
+
+- Saving changes to a script that's incompatible with the interpreter can crash the O3DE Editor. [#11747](https://github.com/o3de/o3de/issues/11747)
+
+**Animation**
+
+- Do not include `.` in names for animations in you DCC tool as this can fail at import time. [#12342](https://github.com/o3de/o3de/issues/12342)
```

Comment:
```suggestion
- Do not include `.` in names for animations in your DCC tool, as this can fail at import time. [#12342](https://github.com/o3de/o3de/issues/12342)
```